gusto/linear_solvers.py

```diff
@@ -156,14 +159,26 @@ def V(u):
         dxp = dx(degree=(self.quadrature_degree))
         dS_vp = dS_v(degree=(self.quadrature_degree))
 
+        # introduce density potential temperatures
+        theta_rho = theta
```

Comment:
Actually it isn't as simple as that, as in `forcing.py` we still need to use the original `theta` so don't want it overwritten by `theta/(1+water_total)`. How about just changing the name `theta_rho` to `theta_weighted` or `theta_water`? Variable naming is hard...